docs/src/pages/components/slider/slider.md

```diff
@@ -105,6 +105,12 @@ This might be less confusing to users compared to a change in direction.
 
 {{"demo": "pages/components/slider/VerticalAccessibleSlider.js"}}
 
+## Theming
```

Comment:
Shouldn't the section be right after "## Customized sliders"? Also, if we move with `## Theming` for the other pages, I would propose we simplify the customization header -> `## Customization`.

Reply:
Sounds good.

Reply:
@oliviertassinari is it better if we use the word "styling" rather than "customization"? I feel "customization" is broader. @danilo-leal any thought?

Reply:
We could consider too

```
## Customization

### Styling

### Theming
```

but maybe it's too deep and a flatten structure would help.

Reply:
Yeah, I prefer flatten structure.

```
## Styling

## Theming
```

I think I will keep `Customization` and ask the team about changing to `Styling` so the change does not overload in this PR. @oliviertassinari should I merge the PR?

Reply:
Sounds good. Do you want Danilo to have a look?

Reply:
I don't think that `Theming` should live in isolation from `Customization`. I believe they are about the same thing. You're customizing the components by applying a given theme in them. But, although this discussion is very important, I don't feel like this is the PR/place to discuss it already.

Reply:
@danilo-leal then I will remove them from the theming section and merge until we have a conclusion.